docs/features/job-queue.txt

```diff
@@ -1,103 +1,159 @@
+| ``ID``        | TBD                                                                          |
++---------------+------------------------------------------------------------------------------+
+| ``Type``      | TBD                                                                          |
++---------------+------------------------------------------------------------------------------+
+| ``Job Name``  | TBD                                                                          |
++---------------+------------------------------------------------------------------------------+
+| ``Priority``  | TBD                                                                          |
++---------------+------------------------------------------------------------------------------+
+| ``Submitted`` | TBD                                                                          |
++---------------+------------------------------------------------------------------------------+
+| ``Slots``     | (acquired/needed)                                                            |
++---------------+------------------------------------------------------------------------------+
+| ``Status``    | TBD                                                                          |
++---------------+------------------------------------------------------------------------------+
+| ``User``      | TBD                                                                          |
++---------------+------------------------------------------------------------------------------+
 
 **********************
  Modify the Job Queue
 **********************
 
 The job queue can be changed in the WebUI **Job Queue** section or by using the CLI ``det job
 update`` command. You can make changes on a per-job basis by selecting a job and a job operation.
-Available operations include:
+Operations include:
 
 -  changing job priorities for agent and Kubernetes priority scheduler
 -  changing weights for agent resource manager using the fair share scheduler
 -  changing the order of queued jobs
 -  changing resource pools
 
-WebUI
-=====
+WebUI Method
+============
 
-To modify the job queue in the Webui,
+To use the WebUI to modify the job:
 
 #. Go to the **Job Queue** section.
 #. Find the job to modify.
 #. Click the three dots in the right-most column of the job.
 #. Find and click the **Manage Job** option.
 #. Make the change you want on the pop-up page, and click **OK**.
 
-CLI
-===
+CLI Method
+==========
 
-To modify the job queue in the CLI, use the ``det job update`` command. Run ``det job update
---help`` for more information. Example operations:
+To use the CLI to modify the job queue, use the ``det job update`` command. For more information,
+run ``det job update --help`` command.
 
-.. code::
+Examples:
+
+.. code:: bash
 
    $ det job update jobID --priority 10
    $ det job update jobID --resource-pool a100
```

Comment:
changes the resource pool of <jobID> to the resource pool named 'a100'.

Also, maybe we should change the name from 'a100' to 'compute-pool' or something...